__tests__/routes.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const supergoose = require('cf-supergoose');
+const mockRequest = supergoose.server(server);
```

Comment:
'server' is not defined

Reply:
commented out for now, until resolution is determined as group